master/pkg/model/environment_config.go

```diff
@@ -91,3 +92,41 @@ func (r *RuntimeItems) For(deviceType device.Type) []string {
 		panic(fmt.Sprintf("unexpected device type: %s", deviceType))
 	}
 }
+
+// RuntimeItemsMap configures the runtime environment variables as a map.
```

Comment:
question: Wait, so why do we need _both_ the list version and the map version now?

Reply:
Keeping the list version for backward compatibility until we have a good approach for making breaking changes in the experiment config. Long term we definitely want to have only the map version.